chain/x/zoracle/endblock.go

```diff
@@ -84,7 +84,7 @@ func handleEndBlock(ctx sdk.Context, keeper Keeper) sdk.Result {
 		// Must never overflow because we already checked for overflow above with
 		// gasConsumed + request.ExecuteGas (which is >= gasUsed).
 		if overflow {
-			panic(sdk.ErrorGasOverflow{Descriptor: "ExecuteRequest"})
+			panic("zoracle::handleEndBlock: Gas overflow")
```

Comment:
Is `sdk.ErrorGasOverflow` deprecated? What's the reason for this change?

Reply:
It's not. I did the change because I ran into some errors. Not sure why. But I change it back now and it works, so good catch!